src/SDL12_compat.c

```diff
@@ -4097,7 +4103,7 @@ UpdateRelativeMouseMode(void)
             if (MouseInputIsRelative) {
                 /* reset position, we'll have to track it ourselves in SDL_MOUSEMOTION events, since 1.2
                  *  would give you window coordinates, even in relative mode. */
-                SDL_GetMouseState(&MousePositionWhenRelative.x, &MousePositionWhenRelative.y);
+                SDL_GetMouseState(&MousePosition.x, &MousePosition.y);
```

Comment:
This is my bug, not yours, but I think this was supposed to be SDL20_GetMouseState(). Calling the 1.2 shim here just sets MousePosition to itself.

Reply:
Should we change this to `SDL20_GetMouseState()`?

Reply:
oh, yes, I thought we fixed that. Sorry! Done now: 19823493da98acb0e938468daacd40bfdb974b0c